source/swap/test/Swap-unit.js

```diff
@@ -19,7 +20,7 @@ contract('Swap Unit Tests', async accounts => {
   const Jun_06_2017T00_00_00_UTC = 1497052800 //a date later than when ganache started
   const mockSigner = accounts[9]
   const mockSender = accounts[7]
```

Comment:
please can these be numbered in numeric order?
And I think `const sender = accounts...` should remain with the other `accounts` definitions?